samples/image-metadata/src/components/AppView/ConfigurationContent.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Heading, Paragraph } from '@contentful/forma-36-react-components';
+import { Divider } from '../Divider';
+import { styles } from './styles';
+
+export const ConfigurationContent = () => (
+  <>
+    <Heading className={styles.heading}>Getting started</Heading>
+    <Paragraph>
+      The Image Wrapper app is now installed. We have set up a demo content type for you with a
```

Comment:
I suggest saying this instead because a user could come back to this page later to change a config.

```suggestion
      The Image Wrapper app is installed. We have set up a demo content type for you with a
```
Also, I feel like we shouldn't use the word 'demo'. We are installing a content type that is certainly usable for non-demo purposes.